api/users/selectors.py

```diff
@@ -3,3 +3,7 @@
 
 def get_profiles():
     return Profile.objects.select_related("user").all()
+
+
+def get_distinct_profiles():
```

Comment:
`Profile.slug` uses `AutoSlugField` should [already be unique](https://django-extensions.readthedocs.io/en/latest/field_extensions.html#current-database-model-field-extensions)  so using distinct here likely will have no effect.